app/utils/display-converter.js

```diff
@@ -123,7 +123,7 @@ module.exports = function (req, data, template) {
   convertedData.currentService = service
   convertedData.isLive = req.isLive
   convertedData.humanReadableEnvironment = convertedData.isLive ? 'Live' : 'Test'
-  const currentPath = (relativeUrl && url.parse(relativeUrl).pathname.replace(/([a-z])\/$/g, '$1')) || '' // remove query params and trailing slash
+  const currentPath = url ? parseUrlFromReq(req).pathname : ''
```

Comment:
`parseUrlFromReq().pathname` doesn't remove trailing `/` so still need to use `replace`